### ISSUE TITLE
Support for Lenovo Ideapad Gaming 82K2

### ISSUE THREAD

**Issue by InstinctEx** [0xless/battmngr#8]
Hello , I recently bought this laptop and I have different output of dmidecode from this [issue](https://github.com/0xless/battmngr/issues/5) so I opened a new one

Laptop Model : `IdeaPad Gaming 3 15ACH6`
Output of dmidecode : `82K2`

Acpi Dump ( **AFTER** performance mode, balanced mode and quiet mode have been toggled so it sniffs the calls ) : [`https://ghostbin.com/TcDK5/raw`](https://ghostbin.com/TcDK5/raw)
(using ctrl+f I am able to find that they have been called, I will try experimenting with it)

I have highlighted 2 acpi calls : 
`._SB_PCI0LPC0EC0_GZ44..p.Quiet mode.`
`/._SB_PCI0LPC0EC0_GZ44.p.Performance mode.`

EDIT : 

It appears that `/._SB_PCI0LPC0EC0_GZ44` is what controls the mode, trying to find out what values should I use

EDIT 2: 
 I have managed to decode some acpi calls using iasl, I have successfully found the correct values. [`https://ghostbin.com/mGZTY/raw`](https://ghostbin.com/mGZTY/raw)

**Comment by 0xless** [comment]
Hi, thanks for showing interest in my project!

In the past I tried using the same approach you are using and I couldn't find some calls and values known to be working.
If you happen to discover a reliable method to extract interesting calls from the acpi dump it would be awesome!

I saw that you forked the repo, I'd suggest you try working with the experimental support branch since it would be easier to add your own calls. Good luck and let me know how it goes!

**Comment by InstinctEx** [comment]
yeah didn't work out, cant find the correct values at all, i'll keep searching
